src/assets/variables.scss

```diff
@@ -1,5 +1,12 @@
-$bg-button:#ec44f7;
-$border-cl-btn:#C338CC;
-$bg-input: #222831;
-$input-text: #EEEEEECC;
-$border-cl:#EEEEEE21;
+$bg_button:#ec44f7;
+$border_cl_btn:#C338CC;
+$bg_input: #222831;
+$input_text: #EEEEEECC;
+$border_cl:#EEEEEE21;
+$body_bg:#191D23;
+$global_color_primary: #EEEEEE;
```

Comment:
![image](https://user-images.githubusercontent.com/966329/191337657-9e32bb91-c523-477f-b3be-4f8d20b2b979.png)

add this variables without prefix global

then you can reuse variables 
for example : `$bg_button : $color_secondary`